shared/more/style-sheet.render.native.js

```diff
@@ -282,10 +316,19 @@ export default class Render extends Component {
     this.setState({check: next})
   }
 
+  _selectTab (idx: number) {
+    this.setState({tabSelected: [idx === 0, idx === 1, idx === 2]})
```

Comment:
Does this limit a TabBars to three tabs?  (Maybe it's fine if it does, I'm not planning to use it for more.)